cfpb/cfgov-refresh
# removed prototype data from page templates to resolve bugs

@richaagarwal I went and lifted your changes specifically to the page templates from your https://github.com/cfpb/cfgov-refresh/pull/1393 into a separate PR. due to some bugs creeping in with the sidebar-contact-molecule.

@kurtw 

### Testing
- Create individual Pages in wagtail and verify they don't break
 - landing page
 - sublanding page
 - browse page
 - browse filterable page
 - document detail page
 - learn page
